src/console/dlt-convert.c

```diff
@@ -343,7 +343,7 @@ int main(int argc, char *argv[])
 
                         if (end == (file.counter - 1)) {
                             /* Sleep if no new message was received */
-                            sleep(0.1);
+                            usleep(100000);
```

Comment:
Please use nanosleep instead to follow current POSIX standard.

Reply:
Changed